openapi/components/parameters/BulkObjectIdNoPassport.yaml

```diff
@@ -3,7 +3,6 @@ description: The object that contains the DRS object IDs array
 properties:
   bulk_object_ids:
     type: array
-    description: DRS object IDs
```

Comment:
minor question: why was this description deleted?

Reply:
There was a duplicate "description" entry here which caused a silent build failure for our documentation (took a long time for me to find that!!)